### DISCUSSION TITLE
New Github Forum

### DISCUSSION THREAD

**Discussion by scandum** [scandum/tintin#19] [Announcements]
The hosting reliability of sourceforge has been getting worse and worse, so I'm moving discussions to github for now.

**Comment by SlySven** [comment]
Do you have a copy of the php fora backed up somewhere - it seems that there is virtually nothing on the *Wayback* machine and it is frustrating not to be able to access all those years of past contributions...

and, of course the SourceForge site still redirects interested parties to the old Forum address which **404**'s now:
> TinTin++ Mud Client says the best way to get help with its software is by visiting http://tintin.sourceforge.io/forum.

**Comment by scandum** [comment]
Don't think I have a copy lying around. The sourceforge website magically stopped working, I'm kind of hoping it'll magically start working again. I have no idea what the matter is.
